rrfs-test/scripts/link_mpasjedi_expr.sh

```diff
@@ -51,4 +51,6 @@ ln -snf ${RDASApp}/fix/expr_data/${exprname}/bkg/restart.2024-05-27_00.00.00.nc
 ln -snf ${RDASApp}/fix/expr_data/${exprname}/obs/* obs/
 ln -snf ${RDASApp}/fix/expr_data/${exprname}/ens/* ens/
 ln -snf ${RDASApp}/fix/expr_data/${exprname}/ref/* ref/
+ln -snf ${RDASApp}/fix/crtm/2.4.0 crtm_2.4.0
```

Comment:
I suggest we don't use the version information in the run directory.
i.e: `ln -snf ${RDASApp}/fix/crtm/2.4.0 crtm_2.4.0`  will be good.
When we want to use a new CRTM version, we only need to update the link and don't need to update the yaml file.